src/terra-dev-site/contributing/ComponentStandards.e.contributing.md

```diff
@@ -0,0 +1,100 @@
+# Terra UI Component Standards
+
+## Cross-Browser Support
+
+Components should be tested and work in the following browsers.
+
+### Desktop Browsers
+
+| [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" alt="Chrome" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)</br>Chrome | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png" alt="Firefox" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)</br>Firefox | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png" alt="IE / Edge" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)</br>Edge  | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/archive/internet-explorer_9-11/internet-explorer_9-11_48x48.png" alt="IE / Edge" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)</br>IE | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png" alt="Safari" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)</br>Safari |
+| --------- | --------- | --------- | --------- | --------- |
+| last 2 versions | last 2 versions | last 2 versions | IE10, IE11| last 2 versions |
+
+### Mobile Browsers
+
+| [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" alt="Chrome" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)</br>Chrome for Android | [<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari-ios/safari-ios_48x48.png" alt="Safari" width="24px" height="24px" />](http://godban.github.io/browsers-support-badges/)</br>Safari for iOS |
+| --------- | --------- |
+| last 2 versions | iOS >= 10 |
+
+A list of the exact browsers this currently includes can be [found here](http://browserl.ist/?q=iOS+%3E%3D+10%2C+last+2+and_chr+versions%2C+last+2+android+versions%2C+last+2+chrome+versions%2C+last+2+edge+versions%2C+last+2+firefox+versions%2C+last+2+ie+versions%2C+last+2+safari+versions).
+
+## Responsive Support
+
+Components should work responsively without introducing horizontal scrollbars on resize.
+
+Terra components are built following the mobile-first approach and utilize the following media query breakpoints:
+
+ - Default: 0+
```

Comment:
I know our testing breakpoints from from tiny to enormous and sizing differs. I realize these should be different, but I think it might be clearing to follow the same scale here?

Reply:
Can you elaborate on this, not sure I follow?

Reply:
Here we are defining Terra breakpoints as default, tiny, small, medium, large, and huge. 

For [testing viewports](https://github.com/cerner/terra-toolkit/blob/master/config/wdio/services.default-config.js#L33) are defined as tiny, small, medium, large, huge, enormous. 
For example, this seems confusing because in testing, the tiny viewport test width is 470, which falls between default & tiny (0-544). I would think this makes me think this test viewport should be called 'default' instead ? 

I know the goal of these test viewports was to hit between these breakpoints. So 🤷‍♀️ maybe I'm just being confusing.

Reply:
nit: want to add `` around the breakpoint values

Tiny: `544x+`

Reply:
Cleared up doc on this: https://github.com/cerner/terra-ui/pull/108/commits/dfacd2ddc1676c12a00c4a4995583e9cdb57421f